### PR TITLE
Add white furnace testing section

### PR DESCRIPTION
**Pull request by portsmouth**
Addresses https://github.com/AcademySoftwareFoundation/OpenPBR/issues/146.

image

image

### CODE REVIEW

**Comment by portsmouth** [comment]
If anyone can think of other cases that should satisfy a white furnace test, please suggest them here.

File: index.html [hunk 1]
```diff
@@ -1331,6 +1331,27 @@
 A more accurate implementation should in principle account for these effects.
 
 
+White furnace testing
+-------------------------------------
+
+The ground truth appearance should perfectly preserve energy in various configurations, in which case the material will satisfy a "white furnace test", i.e.  the object should disappear when illuminated by uniform background light, and rendered in the limit of a large number of bounces. We list here for convenience the configurations in which such a white furnace should pass, all of which would be useful to check as part of a unit testing framework:
+
+   - A fully metallic base (**`base_metalness`**=1) with white **`base_color`** and white **`specular_color`**.
+   - A fully dielectric base (**`base_metalness`**=0), with white **`specular_color`**=0, including:
```
Comment (virtualzavie) [inline index.html:1340]: What does "white **`specular_color`**=0" mean? Did you intend to write "white **`specular_color`**=(1,1,1)" ?

**Comment by virtualzavie** [comment]
> If anyone can think of other cases that should satisfy a white furnace test, please suggest them here.

- Regardless of opacity.
- Regardless of NDF roughness _and_ anisotropy.

**Review by virtualzavie — APPROVED** [review]
LGTM

**Comment by portsmouth** [comment]
@virtualzavie your article on the importance of the white furnace test is nice btw..  (I noticed it while scanning for ideas on how to phrase this section) https://lousodrome.net/blog/light/2023/10/21/the-white-furnace-test/

**Review by jstone-lucasfilm — APPROVED** [review]
Ship it!

**Pull request merged**